tetrateci/test_1.7.sh

```diff
@@ -0,0 +1,21 @@
+set -e
+
+if [[ ${CLUSTER} == "gke" ]]; then
+  # Overlay CNI Parameters for GCP : https://github.com/tetratelabs/getistio/issues/76
+  pip install pyyaml --user && ./tetrateci/gen_iop.py
+  CLUSTERFLAGS="-istio.test.kube.helm.iopFile $(pwd)/tetrateci/iop-gke-integration.yml"
+fi
+
+go test -tags=integ ./tests/integration/operator/...  -p 1  -test.v
```

Comment:
Can you cross check if the same set of tests work for all distros?

Reply:
yes, I wish they do and it's what I'm testing.

Reply:
Finally, I needed to add [this commit](https://github.com/tetratelabs/istio/pull/20/commits/fd0e42164f364d874086bd5a44cabe62614df6dc) to run some tests depending on the distros.